### PR TITLE
B 21427 update addresses is oconus

### PR DESCRIPTION
**Pull request by TevinAdams**
## Summary
This PR includes the following changes:

1. SQL script to update address records in the addresses table with a is_oconus value
2. Populating the is_oconus value on the addresses table when an address is created
3. Populating the is_oconus value on the addresses table when an address is updated

*At this time AK and HI is considered OCONUS

There are many areas throughout the mymove workflow and office workflow where an address is created and updated. It would be good to go through all workflows and make sure places where an address is updated or created works properly. is_oconus is not nullable so if an address is being created or updated without an is_oconus value it will throw an error.

Is there anything you would like reviewers to give additional scrutiny?

[this article](tbd) explains more about the approach used.

## Verification Steps for the Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Have the Agility acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

- [ ] Has the branch been pulled in and checked out?
- [ ] Have the BL acceptance criteria been met for this change?
- [ ] Was the CircleCI build successful?
- [ ] Has the code been reviewed from a standards and best practices point of view?

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

*Note: AK and HI is considered OCONUS
### How to test migration file
1. Run the migrations
2. Verify that the record in the addresses table have a true or false value for is_oconus
3. If country is US or United States then is_oconus should be false. Anything, including state being AK or HI should make is_oconus true

### How to test on address creation
1. Log into MilMove as a customer
2. Create a new shipment and add the address information
3. Check the addresses table for the address you created and make sure is_oconus value is populated

### How to test on service item creation
1. Log in as Prime
2. Go to a shipment
3. Create a serviceItem on a shipment and add an address
5. Check the addresses table for the address you entered and make sure is_oconus value is populated

### How to test address update
1. Log in as Prime
2. Go to a move
3. Update shipments destination address on a shipment and add an address
5. Check the addresses table for the address you entered and make sure is_oconus value is populated

### Frontend

- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://transcom.github.io/mymove-docs/docs/adrs/Browser-Support/#minimum-browser-requirements) (Chrome, Firefox, Edge).
- [ ] There are no new console errors in the browser devtools.
- [ ] There are no new console errors in the test output.
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.
- [ ] This change meets the standards for [Section 508 compliance](https://www.ssa.gov/accessibility/andi/help/install.html).

### Backend

- [ ] Code follows the guidelines for [Logging](https://transcom.github.io/mymove-docs/docs/getting-started/development/logging).
- [ ] The requirements listed in [Querying the Database Safely](https://transcom.github.io/mymove-docs/docs/backend/guides/golang-guide#querying-the-database-safely) have been satisfied.

#### Any new migrations/schema changes:

- [ ] Follows our guidelines for [Zero-Downtime Deploys](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#zero-downtime-migrations).
- [ ] Have been communicated to #g-database.
- [ ] Secure migrations have been tested following the instructions in our [docs](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#secure-migrations).

### CODE REVIEW

File: migrations/app/schema/20241004203140_update_addresses_is_oconus.up.sql
Comment (traskowskycaci) [inline migrations/app/schema/20241004203140_update_addresses_is_oconus.up.sql:1]: Looks like you need to bring the commit for increasing statement timeout in here:
077890e71d83de0db5377ac8150bd2d2aa324e9a

Comment (TevinAdams) [reply migrations/app/schema/20241004203140_update_addresses_is_oconus.up.sql:1]: Added

**Review by traskowskycaci — APPROVED** [review]
Everything looks like it matches doing a file by file comparison, also checked against integrationTesting directly on a few. Looks good

**Review by cameroncaci — APPROVED** [review]

**Review by danieljordan-caci — APPROVED** [review]
Looks like it's all matching up - thanks for the fun times.

**Pull request merged**